english/marshmallow-attack/make-smores/look-for-chocolate/cut-the-leg/cut-the-leg.cd

```diff
@@ -0,0 +1 @@
+oh my god!!! It hurts
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.